cabal-install/tests/IntegrationTests.hs

```diff
@@ -0,0 +1,276 @@
+-- | Groups black-box tests of cabal-install and configures them to test
```

Comment:
I wonder if the `tasty-golden` package can be used to simplify code in this file.